server/lib/routes/post-metrics.js

```diff
@@ -19,6 +22,8 @@ module.exports = function () {
       var metrics = req.body;
       metrics.agent = req.get('user-agent');
       metricsCollector.write(req.body);
+      // send the metrics body to the StatsD collector for processing
```

Comment:
you know, I was just looking at this code for another PR and noticed line 20/23:

```js
metrics.agent = req.body('user-agent');
```

Then we go and send the un-updated req.body to the metricsCollector anyways.